clustered/client/src/test/java/org/ehcache/clustered/client/internal/store/ClusteredStoreTest.java

```diff
@@ -39,18 +42,27 @@
 import org.terracotta.connection.Connection;
 
 import java.net.URI;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
 import java.util.Properties;
 
 import static org.ehcache.clustered.util.StatisticsTestUtils.validateStat;
 import static org.ehcache.clustered.util.StatisticsTestUtils.validateStats;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.junit.Assert.*;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class ClusteredStoreTest {
 
+  private static final RuntimeException RUNTIME_EXCEPTION = new RuntimeException();
```

Comment:
I would need to double check, but I believe reusing an exception instance means its stack trace will be filled only once, the first time it is thrown (or accessed) ... but after that you will always see the same.